app/models/job.js

```diff
@@ -19,6 +19,7 @@ export default DS.Model.extend({
     return this.get('state') === 'F';
   }),
   outputDir: DS.belongsTo('job-output-dir'),
+  jobErrors: DS.hasMany('job-error'),
```

Comment:
I was going to suggest using simply `errors` as the field name, but now I see that's reserved for errors at the model level. Maybe add a comment here so we remember why this isn't called simply `errors`